README.md

```diff
@@ -1,10 +1,12 @@
 # languageServer
 
-- Clone https://github.com/mindula/lsp-sample and set it up using the given instructions
-- Clone languageServer
+
+- This is modified to support the ballerina language server
```

Comment:
What I meant was this line to be in https://github.com/mindula/vscode-extension-samples/tree/main/lsp-sample README.md.